tests/expr_and_series/clip_test.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+import pytest
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+
+def test_clip(request: Any, constructor: Any) -> None:
+    if "pyarrow_table" in str(constructor):
+        request.applymarker(pytest.mark.xfail)
```

Comment:
Also dask should be marked to fail:

```suggestion
    if "pyarrow_table" in str(constructor) or "dask" in str(constructor):
        request.applymarker(pytest.mark.xfail)
```